parliament/cli.py

```diff
@@ -143,6 +143,11 @@ def main():
         type=argparse.FileType("r"),
         default=sys.stdin,
     )
+    parser.add_argument(
+        "--files",
+        help="Provide a comma-separated list of policies",
+        type=str,
+    )
```

Comment:
**suggestion**: Another option is to make `--file` dest variable a list so it can be used multiple times without changing the interface.